GETTING_STARTED.md

```diff
@@ -0,0 +1,30 @@
+The simplest way to start to use 'AppKid' is an example app like:
+
+https://github.com/smumriak/AppKidDemo
+
+...which you can [after installing dependances] simply git-clone to your local machine, and 'swift run' it.  
+
+git clone https://github.com/smumriak/AppKidDemo 
+cd AppKidDemo
+swift build 
+swift run
+
+This application has AppKit defined as a dependancy in its 'Package.swift', and will auto-download and auto-compile AppKid for you when it is 'swift build' or 'swift run' the first time.
+
+see here:
+https://github.com/smumriak/AppKidDemo/blob/3f617eaa9a69ddf2f0333552fd9ded9f1921aa7b/Package.swift#L10
+https://github.com/smumriak/AppKidDemo/blob/3f617eaa9a69ddf2f0333552fd9ded9f1921aa7b/Package.swift#L20
+
+
+This example app does *not* have a main.swift , and instead uses the @main attribute in Sources/AppDelegate.swift to achieve the same result.  In swift if your project provides main.swift it is assumed as "the main function", but its common for swift apps to be built without one [ using @main ], and the code that *would* have been in main.swift is autogenerated.
+
+for more details, please refer to the swift documentation.
+
+refs: 
+https://stackoverflow.com/questions/67429098/what-is-the-differnce-between-main-and-uiapplicationmain
+
+https://developer.apple.com/documentation/swiftui/app/main()
+
+https://docs.swift.org/swift-book/documentation/the-swift-programming-language/attributes/#main
```

Comment:
Why is stackoverflow is the first link here? Why does it precede the official documentation?

Reply:
done